app/__init__.py

```diff
@@ -68,4 +68,13 @@ def create_app(config_name):
     from bulk import bulk as bulk_blueprint
     app.register_blueprint(bulk_blueprint, url_prefix='/bulk')
 
+    from descriptor import descriptor as descriptor_blueprint
+    app.register_blueprint(descriptor_blueprint, url_prefix='/descriptor')
+
+    from resource import resource as resource_blueprint
+    app.register_blueprint(resource_blueprint, url_prefix='/resource')
+
+    from suggestion import suggestion as suggestion_blueprint
```

Comment:
Hmm I'm not sure that's a good enough reason to not have the former functionality in this blueprint. `@login_required` is specified at the route level and not at the blueprint level, so we can have a mixture of routes with and without `@login_required` in a single blueprint.